javaCreateAccountCore/src/main/java/com/hamdan/forzenbook/java/createaccount/core/viewmodel/JavaCreateAccountViewModel.java

```diff
@@ -0,0 +1,207 @@
+package com.hamdan.forzenbook.java.createaccount.core.viewmodel;
+
+import androidx.lifecycle.ViewModel;
+
+import com.hamdan.forzenbook.java.core.Entry;
+import com.hamdan.forzenbook.java.createaccount.core.domain.usecase.CreateAccountEntrys;
+import com.hamdan.forzenbook.java.createaccount.core.domain.usecase.CreateAccountResult;
+import com.hamdan.forzenbook.java.createaccount.core.domain.usecase.CreateAccountUseCase;
+import com.hamdan.forzenbook.java.createaccount.core.domain.usecase.CreateAccountValidationUseCase;
+import com.hamdan.forzenbook.ui.core.R;
+
+import javax.inject.Inject;
+
+import dagger.hilt.android.lifecycle.HiltViewModel;
+import dagger.hilt.android.scopes.ViewModelScoped;
+import io.reactivex.rxjava3.core.BackpressureStrategy;
+import io.reactivex.rxjava3.core.Flowable;
+import io.reactivex.rxjava3.core.Single;
+import io.reactivex.rxjava3.schedulers.Schedulers;
+import io.reactivex.rxjava3.subjects.BehaviorSubject;
+
+@HiltViewModel
+public class JavaCreateAccountViewModel extends ViewModel {
+
+    private final CreateAccountValidationUseCase createAccountValidationUseCase;
+    private final CreateAccountUseCase createAccountUseCase;
+
+    private final BehaviorSubject<JavaCreateAccountState> _state = BehaviorSubject.createDefault(new JavaCreateAccountState());
+
+    @Inject
+    public JavaCreateAccountViewModel(CreateAccountValidationUseCase createAccountValidationUseCase, CreateAccountUseCase createAccountUseCase) {
+        this.createAccountValidationUseCase = createAccountValidationUseCase;
+        this.createAccountUseCase = createAccountUseCase;
+    }
+
+
```

Comment:
Extra spacing here, run lint

Reply:
ktlin ran on it and didnt find it, I'll try the ./gradlew command more in the future because the commit lint seems inconsistent.